Fenced_Frames_Ads_Reporting.md

```diff
@@ -100,4 +100,50 @@ registerAdBeacon({
 });
 ```
 
+## Support for Attribution Reporting 
+### Goals
+*   While fenced frames still have unrestricted network access and FLEDGE supports event-level reporting, the solution below takes advantage of registerAdBeacon/reportEvent information flow to enable [registering attribution sources](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#registering-attribution-sources). [ARA attribution triggering](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#triggering-attribution) is unchanged for registered FLEDGE impressions.
+*   Improve the ergonomics of triggering ad beacons based on clicks.
+
+**As of writing this, the changes below are not yet implemented in Chrome.**
+
+### registerAdBeacon
+
+The worklet code will be able to register an event called “reserved.top_navigation” via registerAdBeacon. 
+
+```
+registerAdBeacon({
+ 'reserved.top_navigation': 'https://adtech.example/click?buyer_event_id=123',
+});
+```
+
+The new event, if registered, implies that an automatic beacon will be sent by the browser to the registered url when a top-level navigation is invoked from within the fenced frame. This will impact top-level navigation initiated from the fenced frame in the same tab (via [unfencedTop target](https://github.com/WICG/fenced-frame/blob/master/explainer/integration_with_web_platform.md#top-level-navigation)) or in a different tab. Note that this beacon is gated on a transient user activation. More details about the beacon are below.
```

Comment:
```suggestion
The new event, if registered, implies that an automatic beacon will be sent by the browser to the registered URL when a top-level navigation is invoked from within the fenced frame. This will impact top-level navigation initiated from the fenced frame in the same tab (via [unfencedTop target](https://github.com/WICG/fenced-frame/blob/master/explainer/integration_with_web_platform.md#top-level-navigation)) or in a different tab. Note that this beacon is gated on a transient user activation. More details about the beacon are below.
```